libsolidity/interface/FileReader.cpp

```diff
@@ -27,19 +27,22 @@ using solidity::frontend::ReadCallback;
 using solidity::langutil::InternalCompilerError;
 using solidity::util::errinfo_comment;
 using solidity::util::readFileAsString;
+using std::distance;
+using std::equal;
+using std::move;
 using std::string;
```

Comment:
Why have these explicitly and not just import `std` as in other files?

Reply:
This file was already using this convention so I just continued.

But ok, I will switch it to our usual convention.